packages/backend-core/src/util/errors.ts

```diff
@@ -0,0 +1,13 @@
+export class JWTExpiredError extends Error {
+  constructor(message: string) {
+    super(message);
+    this.name = 'JWTExpiredError';
```

Comment:
🔧 When extending JS errors this: `Object.setPrototypeOf(this, JWTExpiredError.prototype);` is usually required.

Reply:
@SokratisVidros 
Great reminder!
This is a critical mishap if the target of our compilation was ES5. Currently we are using the latest target for Node 12 which is ES2019 _(we should upgrade btw to ES2020)._ In this case we are safe.